docs/source/release_notes/release-notes.rst

```diff
@@ -65,6 +65,10 @@ develop
          - Enable garbage collection logging for CircleCI builds.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1398>`__)
 
+    *    - |improved|
+         - Increase minimum Cassandra pool size to better handle bursts in requests.
```

Comment:
should indicate that these are the defaults that were changed and the new values